chapters/annotations.tex

```diff
@@ -1272,6 +1272,66 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 \end{lstlisting}
 \end{example}
 
+A connector component declaration may have the following annotation:
+\begin{lstlisting}[language=modelica]
+annotation(mustBeConnected = "message");
+\end{lstlisting}%
+\annotationindex{mustBeConnected}
+
+If non-empty it indicates that it is an error if the connector is not connected from the outside (for a conditional connector this check is only active if the connector is enabled).
+For an array of connectors it applies separately to each element.
+
+\begin{nonnormative}
+Note that this annotation is only intended for non-causal connectors, see \cref{restrictions-of-connections-and-connectors}.
```

Comment:
Redundant _Note_ inside non-normative section:
```suggestion
This annotation is only intended for non-causal connectors, see \cref{restrictions-of-connections-and-connectors}.
```

The _only intended_ almost sounds normative.  Could _only_ be omitted – it not, there seems to be something normative missing?
```suggestion
This annotation is intended for non-causal connectors, see \cref{restrictions-of-connections-and-connectors}.
```